### ISSUE TITLE
Issue when using prefer-destructuring rule together with stage-1 optional chaining (a.k.a. the elvis operator)

### ISSUE THREAD

**Issue by lehni** [babel/eslint-plugin-babel#133]
I’m using Babel `v7.0.0-beta.34`, stage 1 with `eslint-babel` and the `prefer-destructuring` rule. This appears to be clashing with the so called [elvis operator](https://github.com/tc39/proposal-optional-chaining).
As an example, this shouldn’t trigger an linting error, but it does:

```js
const value = object?.value
```
I've gotten in touch with the [eslint people](https://gitter.im/eslint/eslint?at=5a29833d540c78242d97df2e) as well as the babel folks on Slack, and believe that this is the right place to report the issue.

**Comment by kamronbatman** [comment]
Looking for this to be incorporated too.

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [@babel/eslint-plugin has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
